docs/flowmetric-flows-netobserv-io-v1alpha1.adoc

```diff
@@ -120,6 +120,10 @@ Refer to the documentation for the list of available fields: https://docs.opensh
 | `string`
 | Name of the metric. In Prometheus, it is automatically prefixed with "netobserv_".
 
+| `remap`
+| `object (string)`
+| `remap` allows to use different names for the generated metric labels than the flow fields. Use the origin flow fields as keys, and the desired label names as values.
```

Comment:
```suggestion
| Set the `remap` property to use different names for the generated metric labels than the flow fields. Use the origin flow fields as keys, and the label names that you want as values.
```

Reply:
per peer review feedback, since this is still open I figure we can change it still? https://github.com/openshift/openshift-docs/pull/83184#discussion_r1801548779

Reply:
done